### DISCUSSION TITLE
Notification Alerts exits with Error

### DISCUSSION THREAD

**Discussion by diveshu** [wazuh/wazuh#16075] [Q&A]
Notification / Alerts gets invoked but does not get connected. Exits with this error "Error Sending email to <xxx.xxx.xxx.xxx> (smtp server)". The mail server is available and tested via telnet on port 25. But wazuh alert is not getting executed. Is there any way to debug in any of the configurations?

**Comment by jctello** [comment]
Hi @diveshu,
In order to troubleshoot this I recommend the following steps:
- Test out the mail server
  - if you're running postfix locally simply run: `echo "Test mail from postfix" | mail -s "Test Postfix" -r "you@example.com" you@example.com`
  - Since you can connect via telnet you may test the server with a command such as this:
    `for i in {"HELO localhost","MAIL FROM: example@example.com","RCPT TO: example@example.com","DATA","SUBJECT: Test message","Hello, this is a TEST message, please don't reply. Thank you.",".","QUIT"}; do echo $i; sleep 2; done | telnet 127.0.0.1 25`
             In both cases be sure to modify the sender and recipient as well as the mail server's address of the second command.
- Look for error logs in the mail server (if using postfix then `grep -i error /var/log/maillog`
- Verify if there are any relevant messages from Wazuh's mail daemon: `grep mail /var/ossec/logs/ossec.log`
- Verify that alerts have been triggered which would send an email: `grep '"mail":true' /var/ossec/logs/alerts/alerts.json`
- If the problem persists please share any relevant logs as well as the portion of `/var/ossec/etc/ossec.conf` regarding mail configuration

**Reply by diveshu** [reply]
Thank you for your reply. 
Your steps on telnet gave me a lead to resolve the issue. After configuring the domain to local ip in the /etc/host file, alert worked.

One more query, in case if the smtp service is running on another port eg. 26, how can we configure wazuh to connect on port 26.
Note: No postfix running. I tried to change the smtp port in /etc/service file, even then it was not able to connect.

**Reply by jctello** [reply]
Hi @diveshu , I'm glad the answer was helpful.

Currently Wazuh only supports sending email to port 25 or using a "sendmail-like" binary as specified in the reference for the global [smtp-server](https://documentation.wazuh.com/current/user-manual/reference/ossec-conf/global.html#smtp-server) configuration option. I tried setting a port in the server's address but verified that did not work. 

Possible workarounds to this can be using a proxy (like nginx) or using postfix as a relay.

**Reply by diveshu** [reply]
> Hi @diveshu , I'm glad the answer was helpful.
> 
> Currently Wazuh only supports sending email to port 25 or using a "sendmail-like" binary as specified in the reference for the global [smtp-server](https://documentation.wazuh.com/current/user-manual/reference/ossec-conf/global.html#smtp-server) configuration option. I tried setting a port in the server's address but verified that did not work.
> 
> Possible workarounds to this can be using a proxy (like nginx) or using postfix as a relay.

Got it. Thank you @jctello
